test/adapter/json_api/links_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+LinkPost = Class.new(Model)
+class LinkPostSerializer < ActiveModel::Serializer
+  belongs_to :link_author
+end
+
+LinkAuthor = Class.new(Model)
+class LinkAuthorSerializer < ActiveModel::Serializer
+  links do
+    link :self, href: "//link_author/#{object.id}"
```

Comment:
@steveklabnik You're right. To be honest, I didn't really put a lof of thought into the choice of the actual URI in the test, as it is just an arbitrary string so far.